script/cucumber

```diff
@@ -1,4 +1,3 @@
 #!/usr/bin/env bash
 
-time ruby -S bundle exec cucumber \
-  -f Jekyll::Cucumber::Formatter "$@"
+time ruby -S bundle exec cucumber --format summary
```

Comment:
Just for Git History, could we leave line 3 as is?

```sh
time ruby -S bundle exec cucumber \
  --format summary
```

Reply:
Done.